docker/build-tools/prow-entrypoint.sh

```diff
@@ -27,8 +27,24 @@ function log() {
   { set -x; } 2>/dev/null
 }
 
+function read_gcp_secrets() {
+  { set +x; } 2>/dev/null
+  readarray -t secrets < <(<<<"${GCP_SECRETS}" jq -c '.[]')
+  for item in "${secrets[@]}"; do
+    proj="$(<<<"$item" jq .project -r)"
+    secret="$(<<<"$item" jq .secret -r)"
+    env="$(<<<"$item" jq .env -r)"
+    log "Fetching secret '${secret}' in project '${proj}'"
+    value="$(gcloud secrets versions access latest --secret "${secret}" --project "${proj}")"
```

Comment:
do we need to check if this command succeed? What does it return if the secret is not found?